conda/bootstrap-conda.sh

```diff
@@ -13,7 +13,7 @@ MINICONDA_VARIANT=$(/usr/share/google/get_metadata_value attributes/MINICONDA_VA
 MINICONDA_VERSION=$(/usr/share/google/get_metadata_value attributes/MINICONDA_VERSION || true)
 
 OLD_MINICONDA_VERSION="4.2.12"
-NEW_MINICONDA_VERSION="4.5.4"
+NEW_MINICONDA_VERSION="4.7.12"
```

Comment:
What is the default Python version? It might be a breaking change.

Reply:
Yes, Conda 4.7.10 comes with Python 3.7.3